db/migrate/20190913125905_migrate_subjects.rb

```diff
@@ -0,0 +1,13 @@
+class MigrateSubjects < ActiveRecord::Migration[5.2]
+  def change
+    say_with_time 'populating/migrating course subjects' do
+      all_subject = Subject.all
+      all_course_includes_ucas_subjects = Course.includes(:ucas_subjects, :provider, :subjects)
+      all_course_includes_ucas_subjects.each do |course|
+        course.level = course.ucas_level
```

Comment:
Can we get away without validating `level`? We haven't added any validations for it, yet, I don't think. If so, then we should be able to speed this up a little with:

```suggestion
        course.update_column(:level, course.ucas_level)
```